solarforecastarbiter/metrics/tests/test_preprocessing.py

```diff
@@ -342,6 +342,47 @@ def test_resample_and_align_ref_error_prob(prob_forecasts, single_observation):
             fx_obs, fx_series, obs_series, ref_series, tz)
 
 
+def test_resample_and_align_prob(prob_forecasts, single_observation):
+    tz = 'UTC'
+    fx_obs = datamodel.ForecastObservation(
+        prob_forecasts,
+        single_observation)
+    fx_series = THREE_HOUR_SERIES
```

Comment:
shouldn't this be a dataframe?

Reply:
looks like you're using a dataframe in `test_process_probabilistic_forecast_observations` but I think it would make sense to do it here too in the test with a smaller scope.

Reply:
Good point. Agreed. Although the `prob_forecasts` only has one constant value (0) so I'll just convert the series to a dataframe. i.e., `THREE_HOUR_SERIES.to_frame()`